recipes/anvio-minimal/meta.yaml

```diff
@@ -15,40 +15,34 @@ build:
   script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
-  build:
-    - {{ compiler('c') }}
   host:
-    - python >=3,<3.7
-    - pip
-    - numpy
-    - cython
-    - gsl
+    - python ==3.6
+    - numpy ==1.17.1
+    - setuptools ==41.2.0
+    - wheel ==0.33.6
```

Comment:
why do you need wheel and why in this exact version?

Reply:
The version number does not have to be exact but we pinned it as well with others. I don't know exactly what depends on it but I remember seeing `wheel` many times in `conda-build` conflict logs.